tests/conftest.py

```diff
@@ -67,6 +67,127 @@
     columns=IAMC_IDX + [2005, 2010],
 )
 
+mg_ascen = ['MSG-GLB', 'a_scen']
```

Comment:
I made this comment previously, I think this dataframe is not idea for testing - too large (why multiple regions?), non-standard variables (CO2 emissions from cars and tar), why use `MSG` and `IMG` as model names?

Reply:
My two cents on this: this df is meant for integration tests, not unit tests. Integration tests are always harder, because they’re testing more things but they can catch bugs which unit tests don’t. Of course, totally up to you guys whether you want integration tests and the difficulty they add or not. 

As an integration test, it needs to cover more stuff. Hence multiple regions, weird variables (so you have to think carefully about what is going on, it’s not obvious) and model names that also require some thought.

Reply:
I have no strong opinion here, but would note that there are lots of tests that already depend on these. I *do* have a strong opinion for more tests than fewer. IMO we should spend our efforts primarily on in library code and secondarily on the tests (unless the effort is in making more tests!).

Reply:
Of course I fully agree to have more tests, and also have integration tests with more elaborate datasets. But I also think we should spend a few extra minutes on devising useful datasets for the tests, to avoid situations like #315 (and pending #317) where maintenance becomes difficult (impossible).

My main concern here is not to use "real" model names with made-up data and when using "realistic" variables, stay close to used conventions. 

And @gidden, this test data is currently not used in `master` and the tests in this PR use actual data comparison sparsely, so changing it shouldn't take more than 5 minutes.

Reply:
Ok, I will then ask @znicholls to address this at his earliest convenience

Reply:
Will be a while (weeks to months) before I find time for this. Given this df isn’t in master anymore, I’d suggest writing it however suits the use case you’re testing here, and I’ll deal with bringing something back in when I get to #315.

Reply:
Ok @znicholls - I reimplemented tests using exsiting data in master